.claude/skills/cfn-loop-orchestration/helpers/parse-test-results.sh

```
@@ -0,0 +1,260 @@
+        [[ "$tests_line" =~ ([0-9]+)[[:space:]]*total ]] && total="${BASH_REMATCH[1]}"
+    fi
+
+    [[ "$output" =~ Time:[[:space:]]*([0-9.]+)[[:space:]]*s ]] && duration=$(echo "${BASH_REMATCH[1]} * 1000" | bc | cut -d. -f1)
+
+    while IFS= read -r line; do
+        [[ "$line" =~ ●[[:space:]]*(.*) ]] && failed_names+=("${BASH_REMATCH[1]}")
+    done <<< "$output"
+
+    local pass_rate="0.0000"
+    if [ "$total" -gt 0 ]; then
+        pass_rate=$(printf "%.4f" $(echo "scale=4; $passed / $total" | bc))
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)